### PR TITLE
Modified my README.md file.

### PR DESCRIPTION
**Pull request by Emeziem**
Just successfully learned how to do a pull request from my branch to the main repository.

### CODE REVIEW

**Comment by Emeziem** [comment]
My progress on my portfolio website.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains repo contains Chiemeziem's portfolio and SPS projects.
```
Comment (garron) [inline README.md]: It looks like "repo contains" ended up twice here.

**Review by garron — CHANGES_REQUESTED** [review]
Looks good! One minor thing to address.

File: README.md (hunk 1 above)
Comment (Emeziem) [reply README.md]: I have fixed the redundancy in the README file.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,36 @@
+/* body {
```
Comment (garron) [inline portfolio/src/main/webapp/style.css:1]: While commenting code out as you're working is very useful, I think that for most changes which are sent out for review for a shared project, it's generally preferred to remove commented out code. Thanks to versioning in git, we can always get this back if we need it, so it's not really "lost" by removing it.

This has been pretty universal at the companies that I've worked for. The main exception being things that are commented out very temporarily (e.g. with the intention of disabling some behavior to be soon turned back on).

**Review by garron — APPROVED** [review]
I'm excited to see what it looks like with the new styling!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (Emeziem) [reply portfolio/src/main/webapp/style.css:1]: Understood I will reduce the number of comments in my code next time.

**Pull request merged**